src/demo-app/select/select-demo.html

```diff
@@ -144,6 +144,59 @@
     </mat-card-content>
   </mat-card>
 
+  <mat-card class=".demo-appearance-comparison">
```

Comment:
The dot in the class name looks like it shouldn't be there. Also it doesn't look like this class is being used anywhere.